app/services/translation.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+class Translation
+  # Bei Auswahl der Kategorien „other“ und „discount“ wird hier die Kategorie frei benannt:
+  # category wird dann gesetzt durch den Wert von categorytext
+  def self.price_category(price_category, category_text)
+    case price_category
```

Comment:
wenn `price_category` `nil` ist, dann returned `category_name_for_price` auch implizit `nil`, richtig?
überlege, ob es besser ist, das explizit zu machen?

Reply:
hier im anderen PR in einer `Translation` Klasse machst du es schon explizit: https://github.com/ikuseiGmbH/smart-village-app-converter-xml2json/pull/13/commits/02fcd0105156314b826d2a145386b88f0efae477#diff-d83af154df63d3bbb513b6be842f7246R5

Reply:
Habs geändert

Reply:
Wieso hast du dich hier für nil entschieden und im anderen PR für selbe Logiken für `""`?

Reply:
Weil ich glaube das null eindeutiger als ein leerer string ist. Wenn es Dich stört kann ich es im anderen PR noch ändern.

Reply:
Mich persönlich stört gar nichts. Es betrifft nur Pflege und Verständnis vom Code, der an der Stelle eigentlich gleich ist aber anders. Inkonsequenzen können zu Mehraufwand führen bei Weiterentwicklungen. Deswegen investiere ich die zusätzliche Zeit in PRs auch dafür und weise auf solche Dinge hin.